src/test/java/uk/gov/justice/hmpps/prison/service/DataLoaderRepository.kt

```diff
@@ -48,6 +54,12 @@ class DataLoaderRepository(
   val bedAssignmentHistoriesRepository: BedAssignmentHistoriesRepository,
   val courtEventRepository: CourtEventRepository,
   val offenderRepository: OffenderRepository,
+  val offenderSentenceAdjustmentRepository: OffenderSentenceAdjustmentRepository,
+  val offenderKeyDateAdjustmentRepository: OffenderKeyDateAdjustmentRepository,
+  val offenderPayStatusRepository: OffenderPayStatusRepository,
+  val externalServiceRepository: ExternalServiceRepository,
+  val serviceAgencySwitchesRepository: ServiceAgencySwitchesRepository,
+  val offenderNoPayPeriodRepository: OffenderNoPayPeriodRepository,
```

Comment:
Piggy backing on the existing pattern of a TestGod class to handle all the repositories. I'm not fond of its omnipotence but as a pattern it's working well. Might look into breaking this up, might not.